src/sentry/static/sentry/app/views/acceptOrganizationInvite.tsx

```diff
@@ -76,7 +76,7 @@ class AcceptOrganizationInvite extends AsyncView<Props, State> {
           'Your account ([email]) is already a member of this organization. [switchLink:Switch accounts]?',
           {
             email: user.email,
-            switchLink: <Link href="#" onClick={this.handleLogout} />,
+            switchLink: <Link onClick={this.handleLogout} />,
```

Comment:
I think we should not have a default prop for `to`, it should be required and make sure people thinking about if it should be a link. A lot of times people use a Link when a Button is more appropriate.

That being said this should be `to="#"`